src/Controller/HttpControllerAbstract.php

```diff
@@ -14,102 +17,72 @@
 /**
  * Base class for HTTP controllers.
  */
-abstract class HttpController extends ControllerAbstract
+abstract class HttpControllerAbstract extends ControllerAbstract
```

Comment:
I'm not sure this class is required. The batches will look completely different. Basically all controllers will be http controllers.

Reply:
Ok I renamed the HttpRequest and HttpResponse as well.